packages/components/Button/src/Button.tsx

```diff
@@ -42,7 +41,6 @@ export const Button = compose<ButtonType>({
     root: View,
     icon: Icon,
     content: Text,
-    shadow: Shadow,
```

Comment:
Now that the Shadow component doesn't add any extra views if you don't have a shadow token set.. I think I'd prefer we keep this and actually render the slot in `useRender` below..

Reply:
Is shadow expected to be available for all buttons or just FAB?

Reply:
My assumption was that `shadowToken` is now one of the tokens that clients can configure on a button, even if we don't currently set a default in the default or apple theme. Looking now, I see we never added that token to the spec, or button.types file, so that assumption seems wrong. So I see 2 options:

1) remove the slot altogether, clients can't configure shadows on buttons but I guess they could use the shadow component directly and wrap a button

2) keep the slot, as it doesn't add overhead, and let's client have more customization. 


I don't feel too strongly either way. I do recognize that we currently have a half implementation of shadows in buttons which is bad.

Reply:
I guess also the point of this PR is "remove unused code" not "make unused code work", so we can just remove it